slapp/transfers/upload.py

```diff
@@ -66,13 +72,29 @@ def run(self, db_conn: query_utils.DbConnection):
         uri = utils.s3_uri(self.args['s3_bucket_name'], prefix)
         self.logger.info(f"bucket destination is {uri}")
 
+        # upload the per-experiment object
+        full_video_path = np.unique([m['full-video-source-ref']
+                                     for m in manifests])
+        if len(full_video_path) != 1:
+            raise LabelDataException(
+                    "manifests do not share a single movie path")
+        object_key = prefix + "/" + pathlib.PurePath(full_video_path[0]).name
```

Comment:
I think we should write this using Path objects, makes it more portable.
object_key = `pathlib.PurePath(prefix) / pathlib.PurePath(full_video_path[0]).name`